### DISCUSSION TITLE
Setting entry/exit nodes?

### DISCUSSION THREAD

**Discussion by gbhsrspm** [webfp/tor-browser-selenium#156] [Q&A]
Hi! Newbie question here: when using tor manually, I can set my connections to come from one region by going into the torrc file and setting something like:

EntryNodes {us} StrictNodes 1
ExitNodes {us} StrictNodes 1

I thought the selenium automation would use the same torrc file, but it isn't! And now I don't know how to set those options. Is there a setting I'm missing? I feel like it must be obvious but I'm not seeing it! Thanks, and thanks for making this. :)

**Comment by gunesacar** [comment]
Sorry for the very late answer. This could be achieved by passing the desired parameters in the `torrc` parameter to Stem:
https://github.com/webfp/tor-browser-selenium/blob/main/examples/stem_adv.py#L39-L43
